components/configuration-mgt/org.wso2.carbon.identity.configuration.mgt.endpoint/src/gen/java/org/wso2/carbon/identity/configuration/mgt/endpoint/ResourceApi.java

```diff
@@ -0,0 +1,300 @@
+/*
+ *  Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.wso2.carbon.identity.configuration.mgt.endpoint;
+
+import io.swagger.annotations.ApiParam;
+import org.wso2.carbon.identity.configuration.mgt.endpoint.dto.AttributeDTO;
+import org.wso2.carbon.identity.configuration.mgt.endpoint.dto.ResourceAddDTO;
+import org.wso2.carbon.identity.configuration.mgt.endpoint.dto.ResourceDTO;
+import org.wso2.carbon.identity.configuration.mgt.endpoint.dto.ResourcesDTO;
+import org.wso2.carbon.identity.configuration.mgt.endpoint.factories.ResourceApiServiceFactory;
+
+import javax.ws.rs.Consumes;
+import javax.ws.rs.DELETE;
+import javax.ws.rs.GET;
+import javax.ws.rs.PATCH;
+import javax.ws.rs.POST;
+import javax.ws.rs.PUT;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.Response;
+
+@Path("/resource")
+@Consumes({"application/json"})
+@Produces({"application/json"})
+@io.swagger.annotations.Api(value = "/resource", description = "the resource API")
+public class ResourceApi {
+
+    private final ResourceApiService delegate = ResourceApiServiceFactory.getResourceApi();
+
+    @GET
+
+    @Consumes({"application/json"})
+    @Produces({"application/json"})
+    @io.swagger.annotations.ApiOperation(value = "Retrieve resources.\n", notes = "This API is used to retrieve resources.\n", response = ResourcesDTO.class)
```

Comment:
this line exceeds the 120 characters

Reply:
Intentionally left due to a generated class.